.github/workflows/cd.yml

```diff
@@ -296,6 +298,44 @@ jobs:
         run: |
           npx lerna publish from-package --yes
 
+      - name: pack server visx
+        if: ${{ contains(steps.version-change.outputs.CHANGE, '@microsoft/teamsfx-server') }}
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 10
+          max_attempts: 10
+          retry_on: error
+          command: |
+            sleep 20
+            cd ./packages/server
+            npx vsce package
+
+      - name: release daily server bits to github
+        if: ${{ contains(steps.version-change.outputs.CHANGED, '@microsoft/teamsfx-server') && github.ref == 'refs/heads/dev'  }}
+        uses: ncipollo/release-action@v1.7.3
+        with:
+          token: ${{ secrets.CD_PAT }}
+          tag: 'vscode-extension-${{ steps.version-change.outputs.SERVER_VERSION_NUM }}'
```

Comment:
why the tag has prefix `vscode-extension-` for the release of server bits?

Reply:
because alpha version should not contains ****@, if not, once dev merge to main, the conventional-commits get the wrong start point.

Reply:
thanks! fixed!